scripts/generate-schema/gen-python.ts

```diff
@@ -131,7 +131,7 @@ function generateNode(node: Node, writer: Writer) {
         w(`    ${key}: `);
         if (!required) w("Union[");
         generateNode(value, writer);
-        if (!required) w(" | None] = None");
```

Comment:
Small bug fix. In Python 3.10+ unions are just `|` like in TypeScript (i.e., `T | None`).

If you are using the `typing.Union` type (which is compatible with Python >= 3.8, the syntax is `Union[T, None]`.